content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+# dvc.api.params_show()
+
+Returns all the <abbr>params</abbr> associated with the given <abbr>DVC
+repository</abbr>.
```

Comment:
No, it won't work

Reply:
Not sure about `for the <abbr>stages</abbr>` part, because stages only come into play when using the `stages` argument

Reply:
> it won't work

Is that expected? `dvc params` CLI works with non-Git DVC projects I believe.

Reply:
I misunderstood. I thought you asked about non-dvc projects. 

Non-Git DVC projects do work

Reply:
> dvc params CLI works with non-Git DVC projects I believe.

It doesn't because CLI only has `dvc params diff`, which makes sense to don't be supported in non-Git.
`metrics show` does work and `params show` (if/when added to CLI), should work